.github/workflows/Android-CI-release.yml

```
@@ -19,7 +19,7 @@ jobs:
         with:
           java-version: 1.8
       - name: Install Android SDK
-        uses: hannesa2/action-android/install-sdk@0.0.7.5
+        uses: horie1024/action-android/install-sdk@0.0.8
```

I think the release prefix is missing.

```suggestion
        uses: horie1024/action-android/install-sdk@release/0.0.8
```

---

yes, yes.... I try with prefix